openfecli/commands/plan_rbfe_network.py

```diff
@@ -83,11 +119,22 @@ def plan_rbfe_network_main(
     help=OUTPUT_DIR.kwargs["help"] + " Defaults to `./alchemicalNetwork`.",
     default="alchemicalNetwork",
 )
+@NCORES.parameter(
+    help=NCORES.kwargs["help"],
+    default=1,
+)
+@OVERWRITE.parameter(
+    help=OVERWRITE.kwargs["help"],
+    default=OVERWRITE.kwargs["default"],
+    is_flag=True
+)
 @print_duration
 def plan_rbfe_network(
         molecules: list[str], protein: str, cofactors: tuple[str],
         yaml_settings: str,
         output_dir: str,
+        n_cores: int,
+        overwrite_charges: bool
```

Comment:
I know they should know better but.. I know someone's going to make a mistake here. Should we be extra and write out that they chose this option so it's going to happen?

Reply:
Yeah we can add it to the charge settings that we write out maybe something like this?

```
SMALL MOLECULE PARTIAL CHARGE GENERATOR
_________________________________________

Parsing in Files: 
	Got input: 
		Small Molecules: SmallMoleculeComponent(name=)
Using Options:
	Partial Charge Generation: am1bcc
	Overwriting partial charges

Generating charges: 100%|█████████████████████████| 1/1 [00:00<00:00,  5.21it/s]
	Done
```